<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распознавание размера</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Обозначение:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="designLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPlainTextEdit" name="logPlainText">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="toolMenu">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="runAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="toolMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="runAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="runAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/exec</normaloff>:/exec</iconset>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="connectAction">
   <property name="icon">
    <iconset>
     <normaloff>:/connect</normaloff>:/connect</iconset>
   </property>
   <property name="text">
    <string>Соединение...</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/exit</normaloff>:/exit</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="showQueryAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показать запрос</string>
   </property>
  </action>
  <action name="poolAction">
   <property name="icon">
    <iconset>
     <normaloff>:/pool</normaloff>:/pool</iconset>
   </property>
   <property name="text">
    <string>Пул запросов</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
